### PR TITLE
Mosip-13733 : Added Abis-Handler is Valid route

### PR DESCRIPTION
**Pull request by yaswanths2**

### CODE REVIEW

File: sandbox/registration-processor-mz.properties [hunk 1]
```diff
@@ -446,6 +446,8 @@ registration.processor.policy.id=mpolicy-default-abis
 registration.processor.subscriber.id=mpartner-default-abis
 mosip.regproc.data.share.internal.domain.name=datashare-service
 mosip.regproc.data.share.protocol=http
+mosip.regproc.data.share.modalities.segments={'Finger': {'Left Thumb','Left LittleFinger','Left IndexFinger','Left MiddleFinger','Left RingFinger','Right Thumb','Right LittleFinger','Right IndexFinger','Right MiddleFinger','Right RingFinger'},'Iris':{'Left','Right'},'Face':{'Face'}}
```
Comment (vishwa-vyom) [inline sandbox/registration-processor-mz.properties]: Can you rename this config to mosip.regproc.abis.handler.biometric-modalities-segments-mapping and the move it to the place where all the mosip.regproc.abis.handler. configs are there ? 

Do the same with next config also

**Review by vishwa-vyom — CHANGES_REQUESTED** [review]

File: sandbox/registration-processor-mz.properties [hunk 2]
```diff
@@ -447,7 +447,6 @@ registration.processor.subscriber.id=mpartner-default-abis
 mosip.regproc.data.share.internal.domain.name=datashare-service
 mosip.regproc.data.share.protocol=http
 
-
```
Comment (vishwa-vyom) [inline sandbox/registration-processor-mz.properties]: Please undo this line deletion

File: sandbox/registration-processor-mz.properties [hunk 3]
```diff
@@ -625,6 +624,10 @@ mosip.regproc.abis.handler.server.port=9071
 mosip.regproc.abis.handler.eventbus.port=5726
 mosip.regproc.abis.handler.server.servlet.path=/registrationprocessor/v1/abishandler
 
+mosip.regproc.abis.handler.biometric-modalities-segments-mapping={'Finger': {'Left Thumb','Left LittleFinger','Left IndexFinger','Left MiddleFinger','Left RingFinger','Right Thumb','Right LittleFinger','Right IndexFinger','Right MiddleFinger','Right RingFinger'},'Iris':{'Left','Right'},'Face':{'Face'}}
+mosip.regproc.abis.handler.biometric-segments-exceptions-mapping={'Left Thumb' : 'leftThumb','Right Thumb' : 'rightThumb','Left MiddleFinger' : 'leftMiddle','Left RingFinger' : 'leftRing','Left LittleFinger' : 'leftLittle','Left IndexFinger' : 'leftIndex','Right MiddleFinger' : 'rightMiddle','Right RingFinger' : 'rightRing','Right LittleFinger' : 'rightLittle','Right IndexFinger' : 'rightIndex','Left' : 'leftEye','Right' : 'rightEye','Face' : 'face'}
+
```
Comment (vishwa-vyom) [inline sandbox/registration-processor-mz.properties:629]: Please remove the below 2 additional lines

**Review by vishwa-vyom — CHANGES_REQUESTED** [review]

File: sandbox/registration-processor-mz.properties (hunk 1 above)
Comment (yaswanths2) [reply sandbox/registration-processor-mz.properties]: Incorporated review comments

File: sandbox/registration-processor-mz.properties (hunk 2 above)
Comment (yaswanths2) [reply sandbox/registration-processor-mz.properties]: Undo delted

File: sandbox/registration-processor-mz.properties (hunk 3 above)
Comment (yaswanths2) [reply sandbox/registration-processor-mz.properties:629]: Incorporated review comments

**Review by vishwa-vyom — APPROVED** [review]

**Pull request merged**